styles/styles.css

```diff
@@ -1490,3 +1490,107 @@ main.blue-contract .section.section-with-title p {
   width: 1px;
   white-space: nowrap;
 }
+
+/* generic tooltip styles  starts here */
+.redesign-v2 .tooltip {
+  position: relative;
+}
+
+.redesign-v2 .tooltip::before,
+.redesign-v2 .tooltip::after {
+  position: absolute;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity var(--duration-small) linear;
```

Comment:
also add the transition for `visibility` so it transition out when it is hidden.
`transition: opacity var(--duration-small) linear,visibility var(--duration-small) linear;`